forum/qa-plugin/block-pw/block-pm-override.php

```diff
@@ -22,7 +21,7 @@ function qa_get_request_content(): ?array
     return qa_get_request_content_base();
 }
 
-function qa_user_permit_error($permitoption=null, $limitaction=null, $userlevel=null, $checkblocks=true)
+function qa_user_permit_error(string $permitoption=null, string $limitaction=null, string $userlevel=null, bool $checkblocks=true)
```

Comment:
przeszło Ci to? mi wywalało forum jak tak zrobiłem

Reply:
Działa :v